app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/YouAlreadyHaveEoriController.scala

```diff
@@ -22,8 +22,8 @@ import play.api.mvc._
 import uk.gov.hmrc.eoricommoncomponent.frontend.controllers.auth.AuthAction
 import uk.gov.hmrc.eoricommoncomponent.frontend.domain.LoggedInUserWithEnrolments
 import uk.gov.hmrc.eoricommoncomponent.frontend.models.Service
-import uk.gov.hmrc.eoricommoncomponent.frontend.models.viewModels.StandaloneAlreadyHaveEoriViewModel
```

Comment:
Moved viewModels to other ViewModels package which existed - allowing us to keep these files in one place